### PR TITLE
Add glossary links and modal to Blue Button report generation page

### PR DESCRIPTION
**Pull request by bshyong**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/933

Screenshots:

screen shot 2017-02-07 at 1 40 03 pm

screen shot 2017-02-07 at 1 40 10 pm

### CODE REVIEW

File: src/js/blue-button/components/GlossaryModal.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleCloseModal(event) {
+    event.preventDefault();
+    this.props.onCloseModal();
+  }
+
+  render() {
+    if (this.props.isVisible) {
```
Comment (austin-martinez) [inline src/js/blue-button/components/GlossaryModal.jsx]: Can this be removed? This seems redundant with the visible property of the modal

File: src/js/blue-button/components/GlossaryModal.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Modal from '../../common/components/Modal';
+
+class GlossaryModal extends React.Component {
```
Comment (austin-martinez) [inline src/js/blue-button/components/GlossaryModal.jsx]: Is this component necessary? The isVisible check seems redundant with the Modal component's internal check and use of the visible property, and the close handler could be moved up into the parent component.

**Comment by bshyong** [comment]
Done. A vanilla modal does make sense here. I removed that component in favor of using the common modal that we have.

**Review by austin-martinez — APPROVED** [review]
LGTM :1st_place_medal:

**Pull request merged**